### DISCUSSION TITLE
Breaking changes between 2.2.0 and 2.2.1?

### DISCUSSION THREAD

**Discussion by tylerclark** [capacitor-community/bluetooth-le#544] [General]
I went through the process of upgrading my app to Capacitor 5 and then realized our bluetooth wasn't working well. We could see nearby devices but when we connected to one it would immediately disconnect on `3.0.0` so I reverted down to Cap 4 and realized the following after installing all the versions of this plugin:

`1.8.3` works fine (the one we’ve used for months) :white_check_mark:
`2.0.0` connects but breaks our UI (that's OK) :white_check_mark:
`2.0.1` connects but breaks our UI (that's OK) :white_check_mark:
`2.1.0` connects but breaks our UI (that's OK) :white_check_mark:
`2.2.0` connects but breaks our UI (that's OK) :white_check_mark:
`2.2.1` does not work, connect and immediate disconnect :x:
`2.2.3` does not work, connect and immediate disconnect :x:

I've read the CHANGELOG and README but is there something I'm missing? We cannot go to Cap 5 until we figure out what happened between `2.2.0` and `2.2.1` and beyond. Anyone seen this before? It's very consistent and this is the only plugin I change when I switch between versions.

**Comment by pwespi** [comment]
These are all the changes between 2.2.0 and 2.2.1:
- https://github.com/capacitor-community/bluetooth-le/compare/v2.2.0...v2.2.1

Is this on iOS or also on Android? Are you calling `getConnectedDevices` or `getDevices` right after connecting?
